templates/form.html.twig

```diff
@@ -1,7 +1,11 @@
 {# this tag only applies to the forms defined in this template #}
-{% form_theme form 'bootstrap_4_layout.html.twig' %}
+{% set layout = config.layout.form|default('form_div_layout.html.twig') %}
```

Comment:
Howdy 👋  This PR seems to break a fresh install. If I'm not mistaking, the template `form.html.twig` does not have access to boltform's config. `config` variable is only the general one set in `config/bolt/config.yaml`. This results in a 500 error

<img width="1092" alt="Screenshot 2020-10-24 at 12 36 22" src="https://user-images.githubusercontent.com/1837559/97079604-bef79700-15f5-11eb-9978-08c9cff8181b.png">

Reply:
(actually, line 5 breaks it as it is not `|default`)